config/colao.yml

```diff
@@ -0,0 +1,31 @@
+# PURL configuration for http://purl.obolibrary.org/obo/colao
+
+idspace: COLAO
+base_url: /obo/colao
+
+products:
+- colao.owl: https://raw.githubusercontent.com/insect-morphology/colao/master/colao.owl
+- colao.obo: https://raw.githubusercontent.com/insect-morphology/colao/master/colao.obo
+- colao.json: https://raw.githubusercontent.com/insect-morphology/colao/master/colao.json
```

Comment:
There is no `colao.json` file in the GitHub repository. You can choose to remove this line from the PURL config (easy) or create that file in the repository.